### ISSUE TITLE
Missing pypy.app and Skeinforge.app

### ISSUE THREAD

**Issue by lcis** [repetier/Repetier-Host-Mac#15]
I downloaded the latest source 0.32 and just started build. I'm missing pypy.app and Skeinforge.app.
Where can I get these two apps?

**Comment by repetier** [comment]
Yes these two are missing, because they are only used for packaging to bundle the app with them. It's the skeinforge and pypy folder tree renamed to xy.app. You can also just create a dummy folder with that name. The host itself detects if the folders are there or not. If they are, the host will use them as presets for skeinforge and pypy.
Or you remove them from the build.

**Comment by lcis** [comment]
Ok, Thank you! I created three dummy folders "pypy.app",  "Skeinforge.app" and Slic3r.app".
